From 29d5a8b799e44a37b7ace1487f917f695c02304d Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 23 Feb 2015 22:21:26 +0000
Subject: [PATCH 6/7] lang: Add makepkg (^PKGBUILD, diff, patch, in, install)

Used by the makepkg build system of ArchLinux and MSYS2
---
 src/lang.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lang.c b/src/lang.c
index 35ca2fb..9c402ed 100644
--- a/src/lang.c
+++ b/src/lang.c
@@ -59,6 +59,7 @@ lang_spec_t langs[] = {
     { "lua", { "lua" } },
     { "m4", { "m4" } },
     { "make", { "^Makefile", "^Makefile.Debug", "^Makefile.Release", "Makefiles", "mk", "mak", "make" } },
+    { "makepkg", { "^PKGBUILD", "diff", "patch", "in", "install" } },
     { "mako", { "mako" } },
     { "markdown", { "markdown", "mdown", "mdwn", "mkdn", "mkd", "md" } },
     { "mason", { "mas", "mhtml", "mpl", "mtxt" } },
-- 
2.15.0

